parse-changelog.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -ex
+
+cd "$(dirname "$0")"
+
+docker run --rm \
+  -v "$PWD/.:/work" \
+  -w "/work" \
+  ruby:2.5 bash -ec "
+    gem install -N parse_a_changelog
+    parse ./CHANGELOG.md
+  "
+  
```

Comment:
Github complains about a missing newline here at line 12. Look at the changed files view: https://github.com/cyberark/summon-conjur/pull/47/files

Reply:
I'm not sure how to fix this. There's a new line there. I added another one and it's still complaining